dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/EventExporterTest.java

```diff
@@ -669,6 +669,40 @@ void testOrderEventsOnAttributeDesc()
         assertEquals( List.of( "QS6w44flWAf", "dUE514NMOlo" ), trackedEntities );
     }
 
+    @Test
+    void testOrderEventsOnMultipleAttributesDesc()
+    {
+        EventSearchParams params = new EventSearchParams();
+        params.setOrgUnit( orgUnit );
+
+        params.setFilterAttributes( List.of( queryItem( "toUpdate000" ), queryItem( "toDelete000" ) ) );
+        params.setAttributeOrders( List.of( new OrderParam( "toDelete000", OrderParam.SortDirection.DESC ),
+            new OrderParam( "toUpdate000", OrderParam.SortDirection.DESC ) ) );
+
+        List<String> trackedEntities = eventService.getEvents( params ).getEvents().stream()
+            .map( Event::getTrackedEntityInstance )
+            .collect( Collectors.toList() );
+
+        assertEquals( List.of( "QS6w44flWAf", "dUE514NMOlo" ), trackedEntities );
+    }
+
```

Comment:
if the order is no relevant, as @Ivo is suggesting we should do like https://github.com/dhis2/dhis2-core/pull/11736 and use assertContainsOnly ?

Reply:
The order is exactly what is tested here so I cannot use `assertContainsOnly`